maps/~map_system/maps.dm

```diff
@@ -110,6 +110,7 @@ var/list/all_maps = list()
 
 	var/council_email = "city-council@nanotrasen.gov.nt"
```

Comment:
I assume these emails can't be changed, in which case all should be `var/static/`

Reply:
In future map iterations I want to make it possible to have nanotrasen or non-nanotrasen maps, different colonies perhaps. These emails are created in memory on roundstart.